### PR TITLE
Fixed bug #1604. Check for word FAILURES in comment before truncating

### PR DESCRIPTION
**Pull request by gshimansky**
Signed-off-by: Gregory Shimansky <gregory.shimansky@intel.com>

## What do these changes do?

- [x] passes `flake8 modin`
- [x] passes `black --check modin`
- [x] signed commit with `git commit -s` 
- [x] Resolves #? 
- [x] tests added and passing

### CODE REVIEW

File: ci/teamcity/comment_on_pr.py [hunk 1]
```diff
@@ -50,11 +51,15 @@
         )
         for i in content.split("+ python3 -m pytest ")
     )
+    tests_failed = tests_failed or ("FAILURES" in full_comment)
```
Comment (anmyachev) [inline ci/teamcity/comment_on_pr.py:54]: why not like this?

```suggestion
    tests_failed = "FAILURES" in full_comment
```

Comment (gshimansky) [reply ci/teamcity/comment_on_pr.py:54]: Because it is a loop over `pytest_outputs`.

Comment (anmyachev) [reply ci/teamcity/comment_on_pr.py:54]: I see

**Comment by anmyachev** [comment]
@gshimansky LGTM

**Pull request merged**